wazimap_ng/datasets/admin/forms/indicator_drector_admin_form.py

```diff
@@ -0,0 +1,6 @@
+from django import forms
+from ... import models
+
+class IndicatorDirectorForm(forms.Form):
+    datasetfile = forms.ModelChoiceField(queryset=models.DatasetFile.objects.all(), required=True)
```

Comment:
```suggestion
    dataset_file = forms.ModelChoiceField(queryset=models.DatasetFile.objects.all(), required=True)
```
So a single dataset file can be used in multiple directors?